lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -226,6 +226,11 @@ sub _schedule_iso {
     my $obsolete           = delete $args->{_OBSOLETE}                 // 0;
     my $onlysame           = delete $args->{_ONLY_OBSOLETE_SAME_BUILD} // 0;
     my $skip_chained_deps  = delete $args->{_SKIP_CHAINED_DEPS}        // 0;
+    if (($deprioritize || $obsolete) && $args->{TEST}) {
+        return {error => 'One must not specify TEST and _DEPRIORITIZEBUILD=1/_OBSOLETE=1 at the same time as it is'
+              . 'likely not intended to deprioritize the whole build when scheduling a single scenario.'
```

Comment:
If a complete build is scheduled and then one wants to schedule individual tests but prioritize them over the build I think test+deprio still makes sense, don't you think?

Reply:
I would leave that to the users to decide, see discussion in Rocket Chat.

Reply:
sorry, I don't understand. How can users decide if you prevent that case with an error?

Reply:
just one small question: users could use `_FORCE_OBSOLETE` or `_FORCE_DEPRIORITIZEBUILD` to obsolete or deprioritize when scheduling jobs, why we provide two different parameters? one should work, right?

Reply:
Yes, one would work. I was just following @okurz 's idea because I also thought it makes sense to have one force option matching each setting.